0xPolygon/polygon-edge
# [JSONRPC] Fix websocket subscription

Fixes EDGE-387

# Description

This PR fixes the issue of websocket subscriptions becoming unresponsive after a predefined timeout. 

# Changes include


# Checklist


## Testing


### Manual tests

Use the `websocat` tool to verify the subscription is sending responses:
1. Start a 4-node cluster
2. Connect to a node using `websocat [node_ws_url]`
3. Send the request (in `websocat`):
`{"id": 1, "method": "eth_subscribe", "params": ["newHeads"]}`
4. Observe the responses are being received (subscription is not becoming silent)
